### ISSUE TITLE
Add control of private data

### ISSUE THREAD

**Issue by COLABORATI** [dear-github/dear-github#189]
Github is a privacy desaster. I can not control publishing of my private data, everything I do on github is published without even asking me. Please change that.

**Comment by Burnett01** [comment]
Exactly! I have noticed the same. Github and privacy is a complete mess.
